### ISSUE TITLE
Error when using mim list

### ISSUE THREAD

**Issue by AronLin** [open-mmlab/mim#71]
**Describe the bug**
There is an error when using commad `mim list`.
The version of `mim` is 0.1.3.

**Reproduction**

1. What command or script did you run?

```
    mim list
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?
    No

**Environment**

1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment information and paste it here.

```
sys.platform: linux
Python: 3.7.0 (default, Oct  9 2018, 10:31:47) [GCC 7.3.0]
CUDA available: False
GCC: gcc (GCC) 5.4.0
PyTorch: 1.8.0+cu111
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.7.0 (Git Hash 7aed236906b1f7a05c0917e5257a1af05e9ff683)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.8.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON,

TorchVision: 0.9.0+cu111
OpenCV: 4.5.2
MMCV: 1.3.9
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: not available
MMDetection: 2.14.0+1419e93
```
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [pip]
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**

```
Traceback (most recent call last):
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/bin/mim", line 8, in <module>
    sys.exit(cli())
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mim/commands/list.py", line 27, in cli
    table_data = list_package(all=all)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mim/commands/list.py", line 55, in list_package
    installed_path = get_installed_path(pkg_name)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mim/utils/utils.py", line 77, in wrapper
    return func(package)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mim/utils/utils.py", line 318, in get_installed_path
    return osp.join(pkg.location, package2module(package))
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mim/utils/utils.py", line 77, in wrapper
    return func(package)
  File "/mnt/lustre/linguangchen/anaconda3/envs/pt1.8/lib/python3.7/site-packages/mim/utils/utils.py", line 295, in package2module
    highlighted_error(f'can not infer the module name of {package}'))
ValueError: can not infer the module name of threadpoolctl
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by zhouzaida** [comment]
hi @AronLin , thanks for your feedback. Could you provide the output of `pip list`?

**Comment by AronLin** [comment]
pip list:
```
Package                Version             Location
---------------------- ------------------- --------------------------------------------------
addict                 2.4.0
albumentations         0.5.2
appdirs                1.4.4
certifi                2020.12.5
charset-normalizer     2.0.3
cityscapesScripts      2.2.0
click                  7.1.2
colorama               0.4.4
coloredlogs            15.0
cycler                 0.10.0
Cython                 0.29.23
decorator              4.4.2
humanfriendly          9.1
idna                   3.2
imagecorruptions       1.1.2
imageio                2.9.0
imgaug                 0.4.0
importlib-metadata     4.6.1
joblib                 1.0.1
kiwisolver             1.3.1
Markdown               3.3.4
matplotlib             3.4.2
mmcv-full              1.3.9
mmdet                  2.14.0              /mnt/lustre/linguangchen/codes/test_pr/mmdetection
model-index            0.1.11
networkx               2.5.1
ninja                  1.10.0.post2
numpy                  1.20.3
onnx                   1.9.0
onnxruntime            1.5.1
opencv-python          4.5.2.52
opencv-python-headless 4.5.2.52
openmim                0.1.3
ordered-set            4.0.2
pandas                 1.1.5
panopticapi            0.1
Pillow                 8.2.0
pip                    21.1.1
protobuf               3.17.0
pycocotools            2.0.2
pyparsing              2.4.7
pyquaternion           0.9.9
python-dateutil        2.8.1
pytz                   2021.1
PyWavelets             1.1.1
PyYAML                 5.4.1
requests               2.26.0
scikit-image           0.18.1
scikit-learn           0.24.2
scipy                  1.6.3
setuptools             52.0.0.post20210125
Shapely                1.7.1
six                    1.16.0
sklearn                0.0
tabulate               0.8.9
terminaltables         3.1.0
threadpoolctl          2.1.0
tifffile               2021.4.8
torch                  1.8.0+cu111
torchaudio             0.8.0
torchvision            0.9.0+cu111
tqdm                   4.60.0
typing                 3.7.4.3
typing-extensions      3.10.0.0
urllib3                1.26.6
wheel                  0.36.2
yapf                   0.31.0
zipp                   3.5.0
```

The result of `mim list --all` :

```
 Package                 Version
----------------------  -------------------
zipp                    3.5.0
yapf                    0.31.0
wheel                   0.36.2
urllib3                 1.26.6
typing                  3.7.4.3
typing-extensions       3.10.0.0
tqdm                    4.60.0
torchvision             0.9.0+cu111
torchaudio              0.8.0
torch                   1.8.0+cu111
tifffile                2021.4.8
threadpoolctl           2.1.0
terminaltables          3.1.0
tabulate                0.8.9
sklearn                 0.0
six                     1.16.0
Shapely                 1.7.1
setuptools              52.0.0.post20210125
scipy                   1.6.3
scikit-learn            0.24.2
scikit-image            0.18.1
requests                2.26.0
PyYAML                  5.4.1
PyWavelets              1.1.1
pytz                    2021.1
python-dateutil         2.8.1
pyquaternion            0.9.9
pyparsing               2.4.7
pycocotools             2.0.2
protobuf                3.17.0
pip                     21.1.1
Pillow                  8.2.0
panopticapi             0.1
pandas                  1.1.5
ordered-set             4.0.2
openmim                 0.1.3
opencv-python           4.5.2.52
opencv-python-headless  4.5.2.52
onnxruntime             1.5.1
onnx                    1.9.0
numpy                   1.20.3
ninja                   1.10.0.post2
networkx                2.5.1
model-index             0.1.11
mmcv-full               1.3.9
matplotlib              3.4.2
Markdown                3.3.4
kiwisolver              1.3.1
joblib                  1.0.1
importlib-metadata      4.6.1
imgaug                  0.4.0
imageio                 2.9.0
imagecorruptions        1.1.2
idna                    3.2
humanfriendly           9.1
decorator               4.4.2
Cython                  0.29.23
cycler                  0.10.0
coloredlogs             15.0
colorama                0.4.4
click                   7.1.2
cityscapesScripts       2.2.0
charset-normalizer      2.0.3
certifi                 2020.12.5
appdirs                 1.4.4
albumentations          0.5.2
addict                  2.4.0
mmdet                   2.14.0
```

**Comment by zhouzaida** [comment]
closed by #72
